.changeset/flat-actors-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/link": patch
+---
+
+Set the DisplayName and **ui** of the Link component
```

Comment:
For unique terms, wrap the word with "`". For example, "`__ui__`".

Reply:
fixed
[0b5592f](https://github.com/yamada-ui/yamada-ui/pull/3034/commits/0b5592f5037d778a1a608a9c991acdfaa1f7bf41)